General.ruleset

```diff
@@ -0,0 +1,115 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<RuleSet Name="General C# rules" ToolsVersion="16.0">
```

Comment:
Disable https://rules.sonarsource.com/csharp/RSPEC-3626 too, please. It suggested three items but all three were wrong and would break the code.

Reply:
Disabled.